docs/dockerhub.md

```diff
@@ -21,16 +21,16 @@ docker pull checkmarx/kics:latest
 Scan a directory
 
 ```sh
-docker run -t -v {path_to_host_folder_to_scan}:/path checkmarx/kics:latest scan -p "/path" -o "/path/"
+docker run -t -v {path_to_host_folder_to_scan}:/path checkmarx/kics:latest scan -p /path -o "path_to_results_file"
```

Comment:
I'm not sure this `path_to_results_file` is clear for the user perspective...
The `/path` is more clear since it is mapped to a host dir.

A suggestion is that below (line 33) you clarify that `paths_to_results_file` is based on `/path`
I know it doesn't have to be, but to access it outside the docker env it becomes more clear.

Also, this is not aligned with the other usages we have in the documentation site